src/core/core.element.js

```diff
@@ -1,5 +1,4 @@
-'use strict';
-
+// @ts-nocheck
```

Comment:
Why this?

Reply:
Oh, forgot that there. I enabled esnext for doing some neat stuff, but as it produced around 5k lint errors, left the new stuff out for now. So this is not needed, at least not yet :)